sympy/physics/control/lti.py

```diff
@@ -1300,6 +1300,46 @@ def to_expr(self):
             return Pow(self.den, -1, evaluate=False)
 
 
+class PIDController(TransferFunction):
```

Comment:
Register the class at `sympy/core/tests/test_args.py` to correct the failing optional dependency check.